examples/README.md

```diff
@@ -0,0 +1,41 @@
+# Beangulp Example
+
+This directory contains an example for how to organize yourself and your files
+to automate your Beancount ledger update. See
+http://furius.ca/beancount/doc/beangulp for a fuller documentation and
+discussion of these files.
+
+There are five directories demonstrated here:
+
+* `ledger`: This is a directory, typically a repository under source control,
+  which contains your Beancount ledger file(s) (e.g., `ledger.beancount`) and
+  import configuration ('ledger.import'), which is a Python file.
+
+* `documents`: This is directory which contains your past imported files.
```

Comment:
"`documents`: a directory which contains..." or "documents`: a directory containing..."

Reply:
Done

Reply:
Forgot to commit?